documentation/src/main/asciidoc/topics/con_performance_clustered_cache_modes.adoc

```diff
@@ -99,6 +99,18 @@ However, {brandname} can detect node crashes and update cluster topology changes
 Using asynchronous replication does not guarantee improved throughput for writes, because asynchronous replication limits the number of backup writes that a node can handle at any time to the number of possible senders (via JGroups per-sender ordering).
 Synchronous replication allows nodes to handle more incoming write operations at the same time, which in certain configurations might compensate for the fact that individual operations take longer to complete, giving you a higher total throughput.
 
+[IMPORTANT]
+====
+The JGroups thread pool contains two thread pools: a primary thread pool that creates threads and a secondary thread pool that processes credit requests.
+No limit exists for how many credit requests the secondary thread pool can generate. 
+The primary thread pool can generate a maximum of `200` threads by default. 
+
+Both default thread pool configurations can cause timeout issues for a server installation that requires a high number of concurrent operations for scale-up purposes.
```

Comment:
I am not a fan of this or the preceding paragraph. It immediately starts out sounding like the default values are not sufficient and is in a big IMPORTANT block.

Normally if the JGroups thread pool of 200 threads fills up there is usually a bug somewhere. Although it is normally credit based, but the ways to fix it are not as simple as this and the following paragraph make it seem.

I would personally say to remove this and the last paragraph here and instead just say this above

```
The primary thread pool may use up to a maximum of `200` threads by default. Credit requests may spawn additional threads as needed outside of this thread pool to avoid blocking these requests.
```
until @belaban can confirm what he thinks is best to put here.

Reply:
Updated for now, unless @belaban has objections?